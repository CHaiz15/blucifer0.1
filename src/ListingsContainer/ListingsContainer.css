.listings-header {
  background: linear-gradient(rgba(0, 0, 0, 0.3), rgba(0, 0, 0, 0.3)), url('../assets/listing-img.png') center;
  width: 100%;
  height: 40vh;
  margin-bottom: 3%;
  display: flex;
  justify-content: center;
  align-items: center;
}

.area-title {
  color: #fff;
  font-size: 3em;
  text-shadow: 0 0 3px #000;
  font-family: 'Montserrat', sans-serif;
}

.listings-pg {
  margin: 8vh 0 0 0;
  display: flex;
  flex-direction: column;
  align-items: center;
}

.listings-container {
  width: 100vw;
}

.listing-count {
  align-self: center;
  border-bottom: black solid 1px;
  width: 94%;
}

.listing-cost {
  margin-top: 50%;
}