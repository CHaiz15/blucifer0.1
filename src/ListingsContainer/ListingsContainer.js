import React from 'react';
import './ListingsContainer.css'
import ListingCard from '../ListingCard/ListingCard.js'

const ListingsContainer = ({ listings }) => {
  const appendedListings = listings.map(area => 
    <ListingCard />)
  return(

  )
}

export default ListingsContainer;
