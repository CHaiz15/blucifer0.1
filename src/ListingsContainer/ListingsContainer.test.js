import React from 'react';
import { render } from '@testing-library/react';
import ListingsContainer from './ListingsContainer';
import { shallow } from 'enzyme'

describe('ListingsContainer', () => {
  it('should be an instance of ListingsContainer component', () => {
    const wrapper = shallow(
        <ListingsContainer
            addSelectedListing={jest.fn()}
            selectedAreaId='590'
            listings={[]}
            areas={[{
              id: 590,
              name: "River North",
            }]}
        />);
      expect(wrapper).toMatchSnapshot();
  });
})