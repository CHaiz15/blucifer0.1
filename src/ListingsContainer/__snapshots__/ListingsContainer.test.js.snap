// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListingsContainer should be an instance of ListingsContainer component 1`] = `
<section>
  <h2>
    0
     places to stay
  </h2>
</section>
`;
